newrelic-php-agent/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM alpine:3.10
+FROM alpine:latest
```

Comment:
@cw-niinuma 
When `latest` is used, the image created depends on the docker build timing.
If you want to use the latest version, please use the latest `xx.xx.xx`.